src/eventer/eventer_jobq.c

```diff
@@ -330,7 +330,7 @@ eventer_jobq_create_internal(const char *queue_name, eventer_jobq_memory_safety_
 
 eventer_jobq_t *
 eventer_jobq_create_backq(const char *queue_name) {
-  return eventer_jobq_create_internal(queue_name, EVENTER_JOBQ_MS_CS, mtev_true);
+  return eventer_jobq_create_internal(queue_name, EVENTER_JOBQ_MS_GC, mtev_true);
```

Comment:
What do these changes to the job queue do?

Reply:
Requires the programmer to manage the critical sections.

It should help performance-wise.

Reply:
Okay, that makes bit more sense now.